Code/GraphMol/Atom.cpp

```diff
@@ -517,6 +517,80 @@ void Atom::invertChirality() {
       break;
   }
 }
+
+namespace MDL {
```

Comment:
Using company names for the namespaces isn't something I'd do.
Let's just use `RDKit::setAtomRLabel()`
If setSupplementalLabel() really is necessary (and I'm not 100% convinced), then please `MolOps::setSupplementalSmilesLabel()`